src/ToastContainer.tsx

```diff
@@ -41,6 +42,13 @@ const propTypes = {
     'bottom-center',
     'bottom-end',
   ]),
+
+  /**
+   * By default the container is rendered with `position-absolute` utility class. Provide a string to use other `position-*` utility classes, or `false` to remove it.
```

Comment:
>or `false` to remove it.

Doesn't this accept only strings? 

I think providing an empty string would achieve what you want here 😄 (also it could be a good idea to add a test for that)

Reply:
Oh forget to update this 🙈 I initially allow `false` to remove the `position-*` class, but I feel it is simpler to just use `"static"` or whatever utility class. Using an empty string should work as well, I'll update it.